lib/systems/default.nix

```diff
@@ -2,4 +2,22 @@ rec {
   doubles = import ./doubles.nix;
   parse = import ./parse.nix;
   platforms = import ./platforms.nix;
+
+  # Elaborate a `localSystem` or `crossSystem` so that it contains everything
+  # necessary.
+  #
+  # `parsed` is inferred from args, both because there are two options with one
+  # clearly prefered, and to prevent cycles. A simpler fixed point where the RHS
+  # always just used `final.*` would fail on both counts.
+  elaborate = args: let
+    final = {
+      # Prefer to parse `config` as it is strictly more informative.
+      parsed = parse.mkSystemFromString (if args ? config then args.config else args.system);
```

Comment:
It's an LLVM target triple. The name is unfortunate but my doing. See the code block in https://nixos.org/wiki/CrossCompiling#Cross-compiling_in_practice for example.